### ISSUE TITLE
genvalidity-typed-uuid test failure

### ISSUE THREAD

**Issue by bergmark** [NorfairKing/typed-uuid#2]
Seen in the nigthly build, probably intemittent as we've had succesful builds until now.

```
Full log available at /var/stackage/work/unpack-dir/.stack-work/logs/genvalidity-typed-uuid-0.0.0.1-test.log

    Data.UUID.Typed
        decode :: Data.ByteString.Lazy.ByteString -> Either String (UUID Int)
          ensures that encode and decode are inverses for "unchecked (UUID Int)"'s FAILED [1]

    Failures:

      src/Test/Validity/Aeson.hs:135:21:
      1) Data.UUID.Typed, JSON (UUID Int) (unchecked), decode :: Data.ByteString.Lazy.ByteString -> Either String (UUID Int), ensures that encode and decode are inverse
s for "unchecked (UUID Int)"'s
           Falsifiable (after 1 test and 47 shrinks):
             00000000-0000-0000-0000-000000000000
           Decoding succeeded, but the decoded value
           00000000-0000-0000-0000-000000000000
           differs from expected decoded value
           00000000-0000-0000-0000-000000000000
           'encode' encoded it to the json
           "\"00000000-0000-0000-0000-000000000000\""
    
      To rerun use: --match "/Data.UUID.Typed/JSON (UUID Int) (unchecked)/decode :: Data.ByteString.Lazy.ByteString -> Either String (UUID Int)/ensures that encode and 
decode are inverses for \"unchecked (UUID Int)\"'s/"
    
    Randomized with seed 700511207
    
    Finished in 6.6567 seconds
    9 examples, 1 failure
```

**Comment by NorfairKing** [comment]
It's not intermittent so much as _fixed_ in the latest version on master.
Do I have to release it for you to get that?

**Comment by bergmark** [comment]
Yeah, we pull from hackage. If this is a bug fix then a release would be nice, if it's just fixing the test suite then you may as well wait and bundle it with the next release.

**Comment by NorfairKing** [comment]
@bergmark It's not just the test sutie, it's an actual bug in genvalidity-typed-uuid as a result of an  upgrade of a dependency. (genvalidity-0.11). I'll make a release.

**Comment by NorfairKing** [comment]
@bergmark I released 0.0.0.2

**Comment by bergmark** [comment]
Cool! should show up in nightly soon.
